scripts/run-data-submission-api.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import shutil
+import subprocess
+from collections import namedtuple
+from os import path
+import requests
+import uvicorn
+from fastapi import FastAPI, BackgroundTasks, File, UploadFile, HTTPException
+from data_processing import utils as process_utils
+
+app = FastAPI()
+
+parser = argparse.ArgumentParser(description='Run Cloudnet data submission HTTP API.')
+parser.add_argument('--config-dir', type=str, metavar='/FOO/BAR',
+                    help='Path to directory containing config files. Default: ./config.',
+                    default='./config')
+ARGS = parser.parse_args()
+
+
+@app.post("/data/{hashSum}")
+async def create_upload_file(background_tasks: BackgroundTasks, hashSum: str,
```

Comment:
`UploadFile` pitäisi toimia nimen omaan myös suurilla tiedostoilla, mutta tätä kannattaa kyllä testata käytännössä.

Reply:
FastApin dokumentaatiosta:
>A file stored in memory up to a maximum size limit, and after passing this limit it will be stored in disk.
Kannattaa vilkaista miten tuota maximum size limitiä voi säätää, ettei defaultina ison fileen uploadaus esim. syö ensin kaikkea saatavilla olevaa muistia ja ala vasta sen jälkeen kirjoittamaan sitä levylle.